run_all_experiments.py

```diff
@@ -198,16 +201,21 @@ def parse_args() -> argparse.Namespace:
                       '--introspector-endpoint',
                       type=str,
                       default=introspector.DEFAULT_INTROSPECTOR_ENDPOINT)
-  parser.add_argument('-g',
-                      '--generate-benchmarks',
-                      help='Benchmarks to create',
-                      type=str)
-  parser.add_argument('-p',
-                      '--projects',
-                      help='Projects to target, comma separated string.',
-                      type=str)
-  parser.add_argument('-m',
-                      '--max-benchmark-targets',
+  parser.add_argument(
+      '-g',
+      '--generate-benchmarks',
+      help=('Generate benchmarks and use those for analysis. This is a string '
+            'of comma-separated heuristics to use when identifying benchmark '
+            'targets. Options available: '
+            f'{", ".join(introspector.get_oracle_dict().keys())}'),
+      type=str)
```

Comment:
nit: [`choices`](https://docs.python.org/3/library/argparse.html#choices) is a simple way to ensure a set of allowed values for the arg.

Reply:
There are some other options missing this, will address in a follow-up